test/conformance/tests/test_networkpool.go

```diff
@@ -97,40 +97,49 @@ var _ = Describe("[sriov] NetworkPool", Ordered, func() {
 			Expect(nodeState.Status.System.RdmaMode).To(Equal(consts.RdmaSubsystemModeShared))
 
 			By("Checking rdma mode and kernel args")
+			cmdlineOutput, _, err := runCommandOnConfigDaemon(testNode, "/bin/bash", "-c", "cat /host/proc/cmdline")
+			Expect(err).ToNot(HaveOccurred())
+
 			output, _, err = runCommandOnConfigDaemon(testNode, "/bin/bash", "-c", "cat /host/proc/cmdline | grep ib_core.netns_mode=0 | wc -l")
 			Expect(err).ToNot(HaveOccurred())
-			Expect(strings.HasPrefix(output, "0")).To(BeTrue())
+			Expect(strings.HasPrefix(output, "0")).To(BeTrue(), fmt.Sprintf("kernel args are not right, printing current kernel args %s", cmdlineOutput))
 
 			output, _, err = runCommandOnConfigDaemon(testNode, "/bin/bash", "-c", "cat /host/proc/cmdline | grep ib_core.netns_mode=1 | wc -l")
 			Expect(err).ToNot(HaveOccurred())
-			Expect(strings.HasPrefix(output, "1")).To(BeTrue())
+			Expect(strings.HasPrefix(output, "1")).To(BeTrue(), fmt.Sprintf("kernel args are not right, printing current kernel args %s", cmdlineOutput))
```

Comment:
can we refactor this to?
```go
Expect(cmdlineOutput).To(ContainSubstring("ib_core.netns_mode=1"))
Expect(cmdlineOutput).ToNot(ContainSubstring("ib_core.netns_mode=0"))
```

The same for the other `/host/proc/cmdline` asssertion

Reply:
you are right less execs!